From bf5a1f5e174fdf9a26f64ad46a2a330b55ec48b6 Mon Sep 17 00:00:00 2001
From: Julian Ospald <hasufell@posteo.de>
Date: Fri, 27 May 2016 23:08:44 +0200
Subject: [PATCH] m4/ax_check_openssl.m4: use AC_CHECK_TOOL for pkg-config
Upstream: pending, https://github.com/peti/autoconf-archive/pull/80

Some distros provide prefixed pkg-config. This patch ensures
the right tool is picked.

See https://www.gnu.org/software/autoconf/manual/autoconf.html#Generic-Programs
for details
---
 m4/ax_check_openssl.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/ax_check_openssl.m4 b/m4/ax_check_openssl.m4
index a87c5a6..7084d44 100644
--- a/m4/ax_check_openssl.m4
+++ b/m4/ax_check_openssl.m4
@@ -51,7 +51,7 @@ AC_DEFUN([AX_CHECK_OPENSSL], [
         ], [
             # if pkg-config is installed and openssl has installed a .pc file,
             # then use that information and don't search ssldirs
-            AC_PATH_PROG([PKG_CONFIG], [pkg-config])
+            AC_CHECK_TOOL([PKG_CONFIG], [pkg-config])
             if test x"$PKG_CONFIG" != x""; then
                 OPENSSL_LDFLAGS=`$PKG_CONFIG openssl --libs-only-L 2>/dev/null`
                 if test $? = 0; then
-- 
2.8.3

